jest.config.js

```diff
@@ -175,6 +175,8 @@ module.exports = {
   // A map from regular expressions to paths to transformers
   transform: {
     '.js$': './jest/esm-transformer.js',
+    '\\.tsx?$': 'ts-jest',
+    '\\.ts?$': 'ts-jest',
```

Comment:
`tsx?` means `"ts"`, and optionally `"x"`. `ts?` means `"t"`, and optionally `"s"`. There's no need for the latter.

```suggestion
    '\\.tsx?$': 'ts-jest',
```